pkg/tracing/init.go

```diff
@@ -41,6 +41,7 @@ func registerRoutes(ctx context.Context, app *bunapp.App) error {
 	api := app.APIGroup()
 
 	api.GET("/traces/:trace_id", traceHandler.FindTrace)
+	api.GET("/traces/:trace_id/json", traceHandler.ShowTraceJSON)
```

Comment:
FYI:

- `/traces/:trace_id` should return protobuf
- `"/traces/:trace_id/json` should return plain json (optional)

(will patch during testing)

Reply:
Okay, we already use `/traces/:trace_id` to find a trace by id so I will have to replace it before we can generate a protobuf response.

We also use subdomains to split APIs, e.g. {otlp,api,api2}.uptrace.dev, but it looks like an overkill here.

Reply:
Correction - the path Tempo API expects is the following and should hopefully have no conflicts:

`/api/traces/:traceId/:json` where the :json param is used to switch the response type.

Reply:
Here's a working reference: https://gist.github.com/lmangani/f9ff4bf962068c84d2f151beeb974532

Reply:
> We also use subdomains to split APIs, e.g. {otlp,api,api2}.uptrace.dev, but it looks like an overkill here.

Maybe overkill, but actually quite smart and making lots of sense in keeping things clean and avoid collisions

Reply:
This is resolved.